config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-  get 'pages/blog'
+  resources :users do
+    resources :posts, only: [:show, :index]
+  end
```

Comment:
- [ ] Kindly make sure to expose only the necessary routes for the "users" and "posts"routes, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20you%20added%20code%20to%20handle%20only%20the%204%20required%20URLs.) requirement.